src/views/Drive.vue

```diff
@@ -54,8 +54,7 @@
 						:file="file"
 						:itemIndex="index"
 					/>
-					<DriveGridDrop v-if="sortedFiles.length==0 && currentDir != null && currentDir.isWritable()">
-						Upload a file by dragging and dropping here or clicking the icon
```

Comment:
Have we lost this text?

Reply:
'Drop files here' is displayed as currently. Text comes from DriveGridDrop.vue